docs/source/index.mdx

```diff
@@ -5,14 +5,13 @@ description: Implement a single data graph across multiple services
 ---
 
 import FederationArchitecture from '../shared/diagrams/federation-architecture.mdx';
-
-> **Implementing services** are now known as **subgraphs**.
->
-> If you find an outdated use of "implementing service," please submit article feedback or a pull request using the links on the right.
+import {
+  ExpansionPanel,
+} from 'gatsby-theme-apollo-docs/src/components/expansion-panel';
 
 To get the most out of GraphQL, your organization should expose a [single data graph](https://principledgraphql.com/integrity#1-one-graph) that provides a unified interface for querying any combination of your backing data sources. However, it can be challenging to represent an enterprise-scale data graph with a single, monolithic GraphQL server.
 
-To remedy this, you can divide your graph's implementation across multiple composable services with **Apollo Federation**:
+To remedy this, you can divide your graph's implementation across multiple back-end services with **Apollo Federation**:
```

Comment:
Possibly? I think what you have is also good, just proposing another option. Subgraph might be unintroduced jargon just yet.

```suggestion
To remedy this, you can divide your graph's implementation across multiple subgraphs with **Apollo Federation**:
```

Reply:
The "possibly?" is commentary directed at my own suggestion, btw.

Reply:
It is just barely unintroduced, but I will fix by introducing just-in-time 😄